styles.css

```diff
@@ -1 +1,90 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+* {
+    margin: 0;
+}
+
+:root {
+    --main-bg-color: #FFEFDB;
+    --container-bg-color: #BA3B45;
+    --main-border-color: #9A303A;
+}
+
+#container {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    gap: 16px 16px;
+    grid-template-areas:
+    "area-a"
+    "area-b"
+    "area-c"
+    "area-d"
+    "area-e"
+    " . "
+    "area-f";
+    background-color: var(--main-bg-color);
+    padding: 16px;
+}
+
+#item-a, #item-b, #item-c, #item-d, #item-e, #item-f {
+    background-color: var(--container-bg-color);
+    border: 2px solid var(--main-border-color);
+    color: white;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items:center;
+    padding: 16px;
+}
+
+#item-a {
+    grid-area: area-a;
+}
+
+#item-b {
+    grid-area: area-b;
+}
+
+#item-c {
+    grid-area: area-c;
+}
+
+#item-d {
+    grid-area: area-d;
+}
+
+#item-e {
+    grid-area: area-e;
+}
+
+#item-f {
+    grid-area: area-f;
+}
+
+/*Alleen op mobile die stippellijn rondom het lege element*/
```

Comment:
De stippellijn zie ik in de uitwerking niet terug komen. Ik vroeg me zelf ook af of het nu echt een stippellijn was of meer een aanduiding van 'er is hier niks'. Jij hebt hem wel netjes gedeclareerd ;)

Reply:
Haha...ja ik ben echt een uur bezig geweest om die stomme stippellijn te maken. Vreselijk! En ik zag hem ook niet, dus volgens mij was het helemaal niet eens de bedoeling :P